cmake/configs/default.cmake

```diff
@@ -163,6 +163,7 @@ hunter_config(dbus VERSION 1.10.0-hunter-4)
 hunter_config(dest VERSION 0.8.0-p4)
 hunter_config(dlib VERSION 19.6-p0)
 hunter_config(doctest VERSION 1.2.0)
+hunter_config(double-conversion VERSION 3.0.0)
```

Comment:
Doesn't belong here. From [README](https://github.com/ingenue/hunter):
```
The only pull requests accepting here is about continuous integration testing
on Travis/AppVeyor, i.e. should only affect .travis.yml or appveyor.yml (in some rare cases jenkins.py)
```

Reply:
I think this is because of a merge. Will try reverting that merge.